docs/fieldnorm/control_of_documents.md

```diff
@@ -1,25 +1,91 @@
 
-Examples are:
+## ISO norm requirements
+
+To allow for a "_Bioinformatician_ friendly" QMS, the following aspects need to be considered and documented :
+
+1. Documents are uniquely identified
+2. Documents are approved for adequacy before issue by authorized personnel who have the expertise and competence to determine adequacy
+3. Documents are periodically reviewed and updated as necessary
+4. Relevant versions of applicable documents are available at points of use and, where necessary, their distribution is controlled
+5. Changes and the current revision status of documents are identified
+6. Documents are protected from unauthorized changes and any deletion or removal
+7. Documents are protected from unauthorized access
+8. The unintended use of obsolete documents is prevented, and suitable identification is applied to them if they are retained for any purpose
+9. One paper or electronic copy of each obsolete controlled document is retained for a specified time period or in accordance with applicable specified requirements
+
+## Candidates for a Bioinformatician friendly QMS
+
+_Bioinformaticians_ create workflows collaboratively on a computer within a git environment. Git is a distributed version control system that tracks changes in any set of computer files. This can be used to track the changes of the code itself, but also in its documentation, which can describe the Standard Operating Procedures (SOP) to use it. Git comes in with a set of features allowing to track issues and their resolution, new release of a workflow and its newest features. As the workflow often get upgraded and updated, their documentation, as well, is susceptible to change. Which is why many chose _Bioinformaticians_ their own form of QMS. Examples of those documentations systems are:
 
 1. Github pages
-2. Gitlab
-3. Confluence
\ No newline at end of file
+2. Gitlab pages
+3. Confluence
+
+Many of those tools, if used in a specific way, can allow _Bioinformaticians_ to work in a comfortable environement, while complying to the ISO norms requirements. They are of course multiple ways to implement such a system, and as long as it is documented how your system complies to the ISO requirements, all is well. If you are looking for some inspirations, in the next paragraph we will describe an example on how to use one this tool to follow the ISO requirements for both controlling the workflows/software you develop, and their associated documentation/SOP.
+
+## Example case : The usage of GitLab
+
+### Central gathering of all SOP on Gitlab pages
```

Comment:
I love this. In the future we should aim to have similar examples for both Confluence and Github pages as well. Not Roxannes task though.

Reply:
Let me know if you want to discuss this topic one day ;)